.moban.yml

```diff
@@ -0,0 +1,7 @@
+configuration:
+  template_dir:
+    - templates
```

Comment:
The recommended approach now is to put templates into a `.moban.dt` .  That is why my PRs do.  We need to choose the same approach , for consistency sake.  c.f. https://github.com/coala/coala/pull/4897/files#diff-ce88a447f57dae1c1ad94204c8eb7dc5R40